### ISSUE TITLE
[MobileNerf] Integrating result to unity or omniverse

### ISSUE THREAD

**Issue by devloper13** [google-research/jax3d#176]
First, my thanks for releasing the MobileNerf code.

This is regarding the polygon rendering method in the paper. Does this also mean that these results can be directly integrated to tools like Unity or Omniverse? Traditionally, the Nerf results are first converted to meshes and then can probably be imported but the result of the mesh is often terrible.

**Comment by czq142857** [comment]
Yes. Some people have already implemented that. For example, https://github.com/julienkay/MobileNeRF-Unity-Viewer
